Makefile

```diff
@@ -278,7 +278,7 @@ integration_all: $(MINISHIFT_BINARY)
 	--run-before-feature="$(RUN_BEFORE_FEATURE)" --test-with-specified-shell="$(TEST_WITH_SPECIFIED_SHELL)" --copy-oc-from="$(COPY_OC_FROM)" $(GODOG_OPTS)
 
 .PHONY: integration_pr ## Run integration tests for pull request (skip more specialized tests)
-integration_pr: GODOG_OPTS = --tags=core\&\&~disabled
+integration_pr: GODOG_OPTS = --tags=addon-che,addon-che-with-anyuid
```

Comment:
@l0rd This needs to change back so that we can merge this PR.

Reply:
I have changed this back @LalatenduMohanty